### DISCUSSION TITLE
How to use the DPA-2 pretrained model for MD simulation

### DISCUSSION THREAD

**Discussion by junqili** [deepmodeling/deepmd-kit#4444] [Q&A]
### Summary

I wonder whether i can use the DPA-2 pretrained model for MD simulation directly. But I only found the model in ".pt" format. I want to ask whether DPA-2 pretrained model can be used for MD, and how to do that. Thank you!

### Detailed Description

I wonder whether i can use the DPA-2 pretrained model for MD simulation directly. But I only found the model in ".pt" format. I want to ask whether DPA-2 pretrained model can be used for MD, and how to do that. Thank you!

**Comment by njzjz** [comment]
You need to use `dp --pt freeze` to generate the frozen model file from the checkpoint file.

**Reply by KK33999** [reply]
May I ask if it is possible to run LAMMPS directly on a model such as model.ckpt-300000.ct in Deepmd kit v3.0.2 without using the dp -- pt freeze command?
